### ISSUE TITLE
[Bug]: Collections for Tonbridge & Malling do not seem to be updating

### ISSUE THREAD

**Issue by sowery** [mampfes/hacs_waste_collection_schedule#1590]
### I Have A Problem With:

A specific source

### What's Your Problem

The Tunbridge and Malling integration / date collection does not seem to be working. I can search the details from the council web page manually but they are not updating beyond the end of this year (2023). 

### Relevant Configuration

```YAML
I have checked the configuration with the details in the instructions. There are no changes that I can see either in my configuration or in the council web pages.
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [ ] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by EPicLURcher** [comment]
I am having the same experience with the West Berkshire council. I have calendar entries up until the end of 2023, but nothing in 2024 (where I expect a collection less than a week from now - but in 2024)

**Comment by Cha7lie** [comment]
I'm having the same issue with West Berkshire council too. I had recycling this week, but my next rubbish is in 2024 and doesn't show.

**Comment by EPicLURcher** [comment]
> I'm having the same issue with West Berkshire council too. I had recycling this week, but my next rubbish is in 2024 and doesn't show.

should we open a separate issue for it? I'm not sure if its a specific source issue or an integration issue

**Comment by 5ila5** [comment]
fixed this bug in #1597 this fix will be part of version 1.45.0 or is available now as master (git) version (HACS: 3-dot menu -> redownload -> change version to master)

but this bug should solve itself in 2024
